### ISSUE TITLE
Right-clicking on search box does not allow paste...

### ISSUE THREAD

**Issue by dmerriman2** [unl/wdntemplates#1268]
It looks like a box I could type into, but no paste option is available when right-clicking on it and so doesn't meet user expectation.  Also since its not an actual form field, the mouse cursor doesn't change to a cursor over it, which again doesn't meet user expectation.
